libs/reports-list/src/lib/reports-list-item/reports-list-item.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="report-item-wrapper">
-  <div>{{ healthCareService }}</div>
-  <div>{{ city }}</div>
-  <div>{{ reportItem?.address }}</div>
+  <div class="healthcare-col">{{ healthCareService }}</div>
+  <div class="city-col">{{ city }}</div>
+  <div class="address-col">{{ reportItem?.address }}</div>
```

Comment:
Please ask Shelly which fields do we need here, now that there's the modal. I think that we don't need address.

Reply:
waiting for an answer from shelly on this

Reply:
according to shelly we can remove the address column